src/main/java/org/folio/clients/InventoryClient.java

```diff
@@ -44,59 +46,50 @@
       return promise.future();
     }
 
-    final var httpClientRequest = buildRequest();
-    final var inventoryClientRequest = httpClientRequest.handler(
-        resp -> {
-          // resp.endHandler(eh -> httpClient.close());
-          final var instances = new ArrayList<InventoryHoldingsAndItems>();
-          final var i = resp.statusCode();
-          if (i != 200) {
+    String inventoryUrl = String.format("%s%s", okapiUrl, VIEW_URI);
+    HttpRequest<Buffer> request = webClient.requestAbs(HttpMethod.POST, inventoryUrl);
+    request.putHeader(OKAPI_HEADER_TOKEN, okapiToken)
+        .putHeader(OKAPI_HEADER_TENANT, tenantId)
+        .putHeader(ACCEPT, APPLICATION_JSON)
+        .putHeader(CONTENT_TYPE, APPLICATION_JSON);
+    
+    final var instances = new ArrayList<InventoryHoldingsAndItems>();
+    JsonParser jp = JsonParser.newParser();
```

Comment:
Personally, I think it's easier to follow the later code if this variable is named `parser` instead of `jp`.

```suggestion
    JsonParser parser = JsonParser.newParser();
```

Reply:
Done.